src/main/java/controller/RacingGame.java

```diff
@@ -0,0 +1,55 @@
+package controller;
+
+import domain.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class RacingGame {
+    private final List<Car> cars;
+    private TryNumber tryNumber;
+
+    public RacingGame(String carNames, int tryNumber) {
+        this(createCars(parseCarNames(carNames)), new TryNumber(tryNumber));
+    }
+
+    public RacingGame(List<Car> cars, TryNumber tryNumber) {
+        this.cars = cars;
+        this.tryNumber = tryNumber;
+    }
+
+    private static List<Car> createCars(List<String> parsedCarNames) {
+        List<Car> cars = new ArrayList<>();
+
+        for (String carName : parsedCarNames) {
+            cars.add(new Car(carName, 0));
```

Comment:
```suggestion
            cars.add(new Car(carName));
```
이와 같이 구현이 가능하도록 Car에 생성자를 추가해 보면 어떨까?